packages/core/src/modules/sd-jwt-vc/SdJwtVcOptions.ts

```diff
@@ -47,7 +47,7 @@ export type SdJwtVcPresentOptions<Payload extends SdJwtVcPayload = SdJwtVcPayloa
   /**
    * Use true to disclose everything
    */
```

Comment:
I thin true was removed, what is now the way to disclose everything vs nothing? Can you update the comment?

Reply:
I'll update it :)
FYI: 

- disclose everything: `presentationFrame: undefined`
- disclose nothing: `presentationFrame: {} // empty object`